android/engine/src/main/java/org/smartregister/fhircore/engine/configuration/ConfigurationRegistry.kt

```diff
@@ -96,6 +103,11 @@ constructor(
       .searchCompositionByIdentifier(appId)
       .also { if (it == null) configsLoadedCallback(false) }
       ?.section
+      ?.filter {
+        it.focus.reference.split("/")[0].let { resourceType ->
```

Comment:
Could have the `"/"` on a constant? It's used multiple times on the code

Reply:
Also, should we check where index 0 has anything after the split?

Reply:
This has been updated. The safe null check has been provided.